solutions/2400-2499/2425-bitwise-xor-of-all-pairs-medium.md

```diff
@@ -0,0 +1,99 @@
+---
+description: 'Author: @ganajayant | https://leetcode.com/problems/bitwise-xor-of-all-pairings'
+---
+
+# 2425 - Bitwise XOR of All Pairings (Medium) 
+
+## Problem Link
+
+https://leetcode.com/problems/bitwise-xor-of-all-pairings/
+
+## Problem Statement
+
+You are given two 0-indexed arrays, `nums1` and `nums2`, consisting of non-negative integers. There exists another array, `nums3`, which contains the bitwise XOR of all pairings of integers between `nums1` and `nums2` (every integer in nums1 is paired with every integer in nums2 exactly once).
+
+Return the bitwise XOR of all integers in `nums3`.
+
+**Example 1:**
+
+```
+Input: nums1 = [2,1,3], nums2 = [10,2,5,0]
+Output: 13
+Explanation: A possible nums3 array is [8,0,7,2,11,3,4,1,9,1,6,3].
+The bitwise XOR of all these numbers is 13, so we return 13.
+```
+
+**Example 2:**
+
+```
+Input: nums1 = [1,2], nums2 = [3,4]
+Output: 0
+Explanation: All possible pairs of bitwise XORs are nums1[0] ^ nums2[0], nums1[0] ^ nums2[1], nums1[1] ^ nums2[0],
+and nums1[1] ^ nums2[1].
+Thus, one possible nums3 array is [2,5,1,6].
+2 ^ 5 ^ 1 ^ 6 = 0, so we return 0.
+```
+
+**Constraints:**
+
+- `1 <= nums1.length, nums2.length <= 10^5`
+- `0 <= nums1[i], nums2[j] <= 10^9`
+
+## Approach 1: XOR Properties
+**Concepts**
+* XOR of even times of a number is zero (0)
+* XOR of odd times of a number is number itself
+
+**Understanding** <br />
+let n = nums1.size() and m = nums2.size();
```

Comment:
in code you are using length but here you use `.size()`. Suggest to use the plain text in case people don't know what `.size()` means. 

i.e.

````
let n be the length of $num1$ and m be that of $nums2$
````